package.json

```diff
@@ -32,6 +32,7 @@
     "@types/mocha": "^2.2.33",
     "@types/node": "0.0.2",
     "@types/sinon": "^2.2.2",
+    "@types/reflect-metadata": "0.0.5",
```

Comment:
why do we need this while original reflect-metadata package provides typings in its own package?